requirements-all.txt

```diff
@@ -14,7 +14,7 @@ azure-storage-blob>=12.5.0
 bokeh>=1.4.0
 cryptography>=3.1
 deprecated>=1.2.4
-dnspython>=2.0.0
+dnspython>=1.16.0
```

Comment:
Is there a reason for this?

Reply:
Because ipwhois requires dnspython<=2.0.0. I fixed the code to work with either pre-2.0.0 or post